docs/examples/bci2000/connect.md

```diff
@@ -0,0 +1,133 @@
+# Connecting DSI Headset
+
+This guide will walk you through how to connect your Wearable Sensing DSI headset to BCI2000 for data acquisition.
+
+```{admonition} Note
+:class: sidebar note
+Please ensure BCI2000 is already installed on your system. If you haven't installed it yet, you can find the download instructions [here](https://www.bci2000.org/mediawiki/index.php/DownloadBCI2000).
+```
+
+## Update the DSI API Driver
+
+BCI2000 includes a community-contributed module called DSISerial, which supports Wearable Sensing DSI EEG systems. This module is already built into BCI2000, but you need to update its driver manually to ensure full compatibility.
+
+1. Download the Compatible Driver File
+
+   * Download the driver that was built alongside the current version of *DSISerial*. Although new versions of the Wearable Sensing *DSI API* exist, they have not yet been tested with *DSISerial* and may not work correctly.
```

Comment:
You only need the .dll file, correct? Why would they download the whole thing? The only thing you need is the dll and to locate the one for your operating system. In this example, we used `libDSI-Windows-x86_64.dll`